Vagrantfile

```diff
@@ -9,63 +9,40 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # options are documented and commented below. For a complete reference,
   # please see the online documentation at vagrantup.com.
 
+  #calculate memory allocation
+
+  hostmem = Integer(%x[ grep MemTotal /proc/meminfo ].scan(/\d+/).shift) / 1024
+  if hostmem >= 8000 then 
+    dev_mem = 4096
+  elsif hostmem < 4000 and hostmem > 2048 then 
```

Comment:
Yes. You are right. The complete block does not make sense. It must be:
>8000 4GB guest memory
between 4000 and 8000 2GB guest memory
<2000 1GB Guest Memory.